src/main/resources/db/changelog/v.0.1/db.changelog-master-v.0.1.yaml

```diff
@@ -0,0 +1,5 @@
+databaseChangeLog:
+  - include:
+      file: db/changelog/v.0.1/db.changelog-internship.yaml
```

Comment:
Enough to add here only fileName because `db.changelog-master-v.0.1.yaml ` is in the same path.

Reply:
we changed as u said but it doesnt work. Only with full filePath

Reply:
Sorry, I missed tag `relativeToChangelogFile`. try with it. It will look like
databaseChangeLog:
  - include:
      file: db.changelog-internship.yaml
      relativeToChangelogFile: true

Reply:
already done